notify/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "notifyms"
+version = "0.1.0"
+description = ""
+authors = ["ZheTao <zacharytao99@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+requests = "^2.28.2"
+pika = "^1.3.1"
+responses = "^0.23.1"
+sendgrid = "^6.10.0"
+python-dotenv = "^1.0.0"
+load-dotenv = "^0.1.0"
```

Comment:
seems like all `load-dotenv` does is [run `load_dotenv()`](https://github.com/fly/load-dotenv/blob/main/load_dotenv.py) and nothing else

shouldn't add it as a dependency here